runbotics/runbotics-orchestrator-ui/src/utils/text.ts

```diff
@@ -1 +1,20 @@
 export const capitalizeFirstLetter = (text: string) => text.charAt(0).toUpperCase() + text.slice(1).toLowerCase();
+
+export const convertToPascalCase = (rawText: string) => {
+    const text = rawText.toLowerCase().replace('_', ' ');
```

Comment:
could be done easier, you can split string elements by spaces and lower cases so it gives you a list of string and just iterate through them and change first letter to upper case and merge them into single string